content/posts/newsletter/2019-11-08.md

```diff
@@ -0,0 +1,125 @@
+
+![Quickblox examples](https://quickblox.com/wp-content/uploads/2019/10/xmpp-body.png)
+
+Martin aka debacle wrote an short piece bout [Dino and other updated software](https://xmpp-team.pages.debian.net/blog/2019/10/dino-with-jet-and-bookmarks2.html) in Debian.
+
+Ben Kwiecien [compared ejabberd and Prosody](https://kwiecien.us/archives/2019/10/16/ejabberd_vs__prosody/index.html) in plain text.
+
+Neetesh Mehrotra wrote about XMPP as a [Communication Protocol for the IoT](https://opensourceforu.com/2019/10/xmpp-a-communication-protocol-for-the-iot/), comparing it to MQTT and HTTP.
+
+Jon Erlichman published a photo of an early sktech of Twitter in 2005 by Jack Dorsey... which features [Jabber at the core](https://twitter.com/JonErlichman/status/1191369676531621888).
+
+![Jack Dorsey's 2005 Twitter](https://pbs.twimg.com/media/EIiYZ8PXkAYuf5Y?format=jpg&name=900x900)
+
+And last but not least, there are translations everywhere! The newsletter for the last month has been translated in [German](https://www.jabber.de/xmpp-newsletter-01-oktober-2019-fosdem-2020-modernisierung-von-xmpp-peer-networks/) and [Spanish](https://www.jabber.de/xmpp-newsletter-01-de-octubre-de-2019-fosdem-2020-modernizacion-de-xmpp-peer-networks/)!
+
+## Tutorials
+
+Erlang Solutions republished their tutorial on how to [Build a complete iOS messaging app using XMPPframework](https://www.erlang-solutions.com/blog/build-a-complete-ios-messaging-app-using-xmppframework-tutorial-part-1.html), with its [part 2](https://www.erlang-solutions.com/blog/xmpp-protocol-build-an-ios-instant-messaging-app-part-2.html).
+
+## Events
+
+The pace of local meetings and sprints is still high within the XMPP community.
+
+pep tells us the story of the "[Sprint in the cold north](https://bouah.net/2019/10/sprint-in-the-cold-north/)", with sauna and crêpes, working on new groupchat bookmarks specification, file transfer interoperability issues, and a future landing page for new XMPP users.
+
+![Stockholm](https://bouah.net/2019/10/stockholm.jpg)
+
+## Software releases
+          
+### Servers
+
+Erlang Solutions has released MongooseIM 3.5.0 ([go check the changelog](https://github.com/esl/MongooseIM/releases/tag/3.5.0)), and published an article on [GDPR in instant messaging](https://www.erlang-solutions.com/blog/mongooseim-designed-with-privacy-in-mind.html).
+
+This month the Ignite Realtime community has released:
+
+* [Openfire 4.4.3](https://discourse.igniterealtime.org/t/openfire-4-4-3-release/)
+*  [inVerse Openfire plugin 5.0.4.1](https://discourse.igniterealtime.org/t/inverse-openfire-plugin-5-0-4-1-released)
+* [User Status Openfire plugin 1.2.2](https://discourse.igniterealtime.org/t/user-status-openfire-plugin-1-2-2-released)
+* [HTTP File Upload plugin 1.1.1](https://discourse.igniterealtime.org/t/http-file-upload-plugin-1-1-1-released/86563)
+
+Marek Foss has announced [https://blog.process-one.net/ejabberd-19-09-1/ ejabberd 19.09.1].
```

Comment:
I think that the markup for the link is wrong (it won't be clickable)

Reply:
Thx